src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Assistant/TicketAssistantTile.tsx

```diff
@@ -17,10 +23,26 @@ export const TicketAssistantTile: React.FC<TicketAssistantProps> = ({
 }) => {
   const styles = useStyles();
   const {themeName} = useTheme();
+  const iconColor: StaticColor = 'background_accent_2';
   const color: StaticColor = accented ? 'background_accent_3' : 'background_0';
   const themeColor = getStaticColor(themeName, color);
   const {t} = useTranslation();
 
+  const {fareProductTypeConfigs, preassignedFareProducts} =
+    useFirestoreConfiguration();
+
+  const sellableProductsInApp = preassignedFareProducts.filter(
+    productIsSellableInApp,
+  );
+
+  const sellableFareProductTypeConfigs = fareProductTypeConfigs.filter(
+    (config) => sellableProductsInApp.some((p) => p.type === config.type),
+  );
+  // Period ticket Config
+  const config = sellableFareProductTypeConfigs.find(
+    (config) => config.configuration.requiresLogin,
+  );
+
```

Comment:
Here you have commented "// Period ticket Config". Not sure what is ment here, but interpreting `requiresLogin=true` as "this is the period product" is kinda brittle. Even though it as of now is true, it is no guarantee that it won't change.